packages/logs/src/domain/configuration.ts

```diff
@@ -28,11 +39,30 @@ export function validateAndBuildLogsConfiguration(
     return
   }
 
+  let forwardConsoleLogs: StatusType[] = []
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    if (
+      initConfiguration.forwardConsoleLogs !== undefined &&
+      initConfiguration.forwardConsoleLogs !== 'all' &&
+      !Array.isArray(initConfiguration.forwardConsoleLogs)
+    ) {
+      display.error('Forward Console Logs should be an array')
+      return
+    }
+
+    forwardConsoleLogs =
+      initConfiguration.forwardConsoleLogs === 'all' ? CONSOLE_APIS : initConfiguration.forwardConsoleLogs || []
+  }
+
+  if (initConfiguration.forwardErrorsToLogs) {
+    forwardConsoleLogs.push(ConsoleApiName.error)
+  }
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** you could remove the need for a `removeDuplicates` function by either:

* testing if the value isn't there yet:
```suggestion
  if (initConfiguration.forwardErrorsToLogs && !includes(forwardConsoleLogs, ConsoleApiName.error)) {
    forwardConsoleLogs.push(ConsoleApiName.error)
  }
```

* using a `Set` for `forwardConsoleLogs` value (also use a `Set` as parameter to `initConsoleObservable`)

Reply:
Actually, it seems like `new Set(arg)` is not supported by IE11 (according to https://kangax.github.io/compat-table/es6/#test-Set ) so maybe use the first solution

Reply:
The first solution does not cover if a user send `forwardConsoleLogs: ['error', 'error']`. Or maybe I'm missing something?

Reply:
Ah right.